lib/utils/network_utils.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:flutter/material.dart';
```

Comment:
`network_utils` is ambiguous. It would infer that we deal with network (IP, TCP...) -related stuff when in reality we are dealing with social networks. Consider renaming the file